server/src/main/java/org/opensearch/index/translog/RemoteFsTranslog.java

```diff
@@ -248,6 +251,11 @@ public void onUploadFailed(TransferSnapshot transferSnapshot, Exception ex) thro
 
     }
 
+    private void logUploadStats(long uploadStartTime, boolean uploadStatus) {
+        logger.debug("Translog Upload status={} timeTaken={}", uploadStatus, (System.currentTimeMillis() - uploadStartTime));
+
+    }
```

Comment:
Always use nano time for measuring elapsed time

Reply:
Incorporated.

Reply:
I think maybe something was missed here. It looks like this class is still calculating relative time using the system clock.

Reply:
this was added for debugging during development. Missed removing this code.